package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "create-abbeys-webpack-initializer",
+  "version": "1.0.3",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest --verbose",
+    "test:watch": "npm run test -- --watch"
+  },
+  "bin": {
+    "create-abbeys-webpack-initializer": "./src/index.js"
+  },
+  "keywords": [],
+  "author": "abbey masters",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.6.4",
```

Comment:
the only dependencies you need for this repo are eslint, jest, and fs - most of these that are installed are only needed for the apps you'll be creating with this script.